targets/UnrealMarketplacePlugin/examplesource/ExampleProject/BuildIOS.sh.ejs

```diff
@@ -1,27 +1,32 @@
 #!/bin/bash
 echo ========== BUILDING MAC ==========
```

Comment:
>MAC [](start = 25, length = 3)

We are building for iOS, should we be more specific here by saying iOS instead of MAC?

Reply:
yes. i'll address it.